src/components/calcite-picker/calcite-picker.tsx

```diff
@@ -0,0 +1,291 @@
+import Sortable from "sortablejs";
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Listen,
+  Method,
+  Prop,
+  State,
+  Watch,
+  h
+} from "@stencil/core";
+import { pencil16 } from "@esri/calcite-ui-icons";
+import CalciteIcon from "../utils/CalciteIcon";
+import guid from "../utils/guid";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-picker",
+  styleUrl: "./calcite-picker.scss",
+  shadow: true
+})
+export class CalcitePicker {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * When true, the rows will be sortable via drag and drop.
+   * Only applies when mode is configuration
+   */
+  @Prop({ reflect: true }) dragEnabled = false;
```

Comment:
can we rename to `draggable`? for dom consistency

Reply:
draggable is a native prop. I had that originally, but I couldn't use it. I forget the error I was getting.

Reply:
gotcha 👍